src/xrpld/ledger/View.h

```diff
@@ -458,6 +530,22 @@ transferXRP(
  */
```

Comment:
I think it's clearer if the docstring above says "if the account lacks required authorization". The current wording leaves me wondering whether the account has the authorization or not.

Reply:
Are you referring to the docstring for `requireAuth()` function?

Reply:
Yes.